legal-api/src/legal_api/models/configuration.py

```diff
@@ -23,6 +24,13 @@
 from .db import db
 
 
+EMAIL_PATTERN = (r'^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|'
+                 r'(".+"))@'
+                 r'((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|'
+                 r'(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$'
+                 )
```

Comment:
This is the pattern that I grabbed from FE, but I'm not sure if we need to validate email pattern before inserting or updating.

Reply:
we can just use this for now